src/test/java/com/wikia/webdriver/testcases/auth/ForcedLoginTests.java

```diff
@@ -5,6 +5,7 @@
 import com.wikia.webdriver.common.core.annotations.Execute;
 import com.wikia.webdriver.common.core.api.ArticleContent;
 import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.logging.Log;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.wikia.webdriver.common.logging.Log'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)